src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -371,6 +380,19 @@ private static bool IsControlBlock(SyntaxNode node)
                 }
             }
 
+            // * { - in property pattern clauses in pattern matching context
```

Comment:
💡 This block can be combined with the block above for `new MyObject {`, `new List<int> {`, and `with {`